mkdocs.yml

```diff
@@ -2,5 +2,28 @@ site_name: Chatterino
 theme:
   name: material
   logo: https://chatterino.com/img/icon-256.png
+  favicon: https://chatterino.com/img/icon-256.png
+  palette:
+    scheme: preference
+  features:
+    - instant
+  icon:
+    repo: fontawesome/brands/github
```

Comment:
Do we need to put a license notice somewhere?

Reply:
Don't think so, these are bundled in with the theme and are free to use. https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#repository-icon
https://github.com/squidfunk/mkdocs-material/blob/master/src/partials/source.html